### PR TITLE
Update ppp docs

### PR DESCRIPTION
**Pull request by ldinghan**

### CODE REVIEW

File: docs/team/ldinghan.md [hunk 1]
```diff
@@ -45,9 +46,11 @@ Given below are my contributions to the project:
   * User Guide:
     * Added documentation for the Session Management Section.
     * Designed the annotated images displaying the different sections of the UI of the product.
-
+    * Reorganised the order of sections in a more logical order.
   * Developer Guide:
     * Designed the class diagram for `Session`.
+    * Designed the sequence diagram for `TakeAttendanceCommand`.
```
Comment (Respirayson) [inline docs/team/ldinghan.md:52]: perhaps could just include the diagram inside here for easy reference

Comment (ldinghan) [reply docs/team/ldinghan.md:52]: sounds like a good idea!

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -360,6 +360,72 @@ This process is summarised in the activity diagram below
...
+This section explains the implementation of the Create Session feature via the `createsession` command.
+The `CreateSessionCommand` causes the specified `Session` to be added to the Session List in the application.
+There are two compulsory fields which are the session number of the session, as well as the names of the students involved.
+
+Below is the activity diagram outlining the execution of `CreateSessionCommand`.
+
+![CreateSessionCommand activity diagram](images/CreateSessionActivityDiagram.png)
+
+Step 1:
+The Avenger(user) enters the command `createsession` and the command is parsed by the `CreateSessionCommandParser`.
```
Comment (wesho1107) [inline docs/DeveloperGuide.md:373]: Perhaps try to standardise across the DG. Were the past instances using "Avenger(user)" as well, or just solely "user" or "Avenger"?

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -360,6 +360,72 @@ This process is summarised in the activity diagram below
...
+
+
+#### Design Considerations:
+**Aspect: How models are modified to store changes**
+
+* **Alternative 1 (current choice):** Cause updates to both `Session` and `Person` when updating the attendance status of a student to a specified session.
+    * Pros: Allows for ease of future enhancements since we can cause the same update from either `Session` or `Person` involved.
+    * Cons: The same action is performed twice, but since the student is added to a `StudentSet`, only one instance of student is added.
+
+* **Alternative 2:** Only update the `Session` to store the `Person`
```
Comment (wesho1107) [inline docs/DeveloperGuide.md:425]: "Causes" and "updates" instead?

**Review by wesho1107 — APPROVED** [review]
Other than a few nits, LGTM!

**Review by raydenlim — APPROVED** [review]
LGTM!

**Pull request merged**